File: app/src/main/java/com/halil/ozel/moviedb/data/models/Episode.java
package com.halil.ozel.moviedb.data.models;

import java.io.Serializable;

public class Episode implements Serializable {
    private Integer episode_number;
    private String name;

    public Integer getEpisode_number() {
        return episode_number;
    }

    public void setEpisode_number(Integer episode_number) {
        this.episode_number = episode_number;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/halil/ozel/moviedb/domain/model/Episode.kt
package com.halil.ozel.moviedb.domain.model

data class Episode(
    val id: Int,
    val name: String,
    val overview: String,
    val stillPath: String?,
    val episodeNumber: Int,
    val seasonNumber: Int,
    val airDate: String,
    val voteAverage: Double,
    val runtime: Int
)
